server/data/quotes.json

```diff
@@ -304,7 +304,7 @@
     "sentence": "Who wants to have an effect on others, must first speak in their language with them."
   },
   {
-	  "author": "A Quote For You",
+	  "author": "Alexis fledge",
```

Comment:
Please capitalize the letter "F" in the word Fledge.

Reply:
Done